src/Microsoft.ML.Data/Transforms/ConvertTransform.cs

```diff
@@ -439,7 +438,7 @@ private static bool TryCreateEx(IExceptionContext ectx, ColInfo info, DataKind k
             // re-fetched by the utils code when needed.
             bool identity;
             Delegate del;
-            if (!Conversions.Instance.TryGetStandardConversion(typeSrc.ItemType, itemType, out del, out identity))
+            if (!Runtime.Data.Conversion.Conversions.Instance.TryGetStandardConversion(typeSrc.ItemType, itemType, out del, out identity))
```

Comment:
How many `Conversions` classes do we have?  This has a bit of a smell to it if we need to use namespaces to specify which one we are talking about.

Reply:
Let's take that separately?
I'll log an issue about inspecting that area. 

---
In reply to: [228949566](https://github.com/dotnet/machinelearning/pull/1328#discussion_r228949566) [](ancestors = 228949566)